### PR TITLE
Make serialization pool in LogDispatcher static to reduece the number of pools

### PR DESCRIPTION
**Pull request by jt2594838**

### CODE REVIEW

**Review by mychaow — APPROVED** [review]
LGTM

**Review by LebronAl — APPROVED** [review]
LGTM

File: cluster/src/main/java/org/apache/iotdb/cluster/log/LogDispatcher.java [hunk 1]
```diff
@@ -68,12 +69,13 @@
   private List<BlockingQueue<SendLogRequest>> nodeLogQueues =
       new ArrayList<>();
   private ExecutorService executorService;
```
Comment (neuyilan) [inline cluster/src/main/java/org/apache/iotdb/cluster/log/LogDispatcher.java:71]: LGTM!
only one question, why only change the _serializationService_ to use google's ThreadFactoryBuilder,
Why not change the _executorService_? Is there any special thinking?

**Review by neuyilan — APPROVED** [review]

File: cluster/src/main/java/org/apache/iotdb/cluster/log/LogDispatcher.java (hunk 1 above)
Comment (jt2594838) [reply cluster/src/main/java/org/apache/iotdb/cluster/log/LogDispatcher.java:71]: Sure I can do this, but I was focusing on serializationService only and ignored executorService. I will modify it too.

Comment (jt2594838) [reply cluster/src/main/java/org/apache/iotdb/cluster/log/LogDispatcher.java:71]: I manually set the thread name in DispatcherThread, so the receiver's name can be included in the thread name, which is not convenient when using ThreadFactoryBuilder.

**Pull request merged**